test/estimation.jl

```diff
@@ -19,3 +19,54 @@ using DSP, Test
     frequencies_estimated = sort(esprit(x, M, p, Fs))
     @test isapprox(frequencies', frequencies_estimated; atol = 1e-2)
 end
+
+@testset "jacobsen" begin
+    # real input
+    fs = 100
+    t = range(0, 5, step = 1/fs)
+    fr = 28.3
+    sr = cos.(2π*fr*t .+ π/4.2)
+    f_est_real = jacobsen(sr, fs)
+    @test isapprox(f_est_real, fr, atol = 0.5)
+    @test jacobsen(zeros(10)) == 0.1
```

Comment:
```suggestion
```
I don't think the frequency estimate of an all-zero signal is something we should make any guarantees about. (But if we do, 0.1 here seems rather surprising.)

Reply:
Again, an effect of the off-by-one bug. This has been fixed.